tests/testthat/test-xnew-value.R

```diff
@@ -1,8 +1,9 @@
 test_that("xnew_value", {
   data <- tibble::tibble(
     a = c(1L, 3L, 4L),
-    b = c(4,4.5,6),
-    d = c("a", "b", "c"))
+    b = c(4, 4.5, 6),
+    d = c("a", "b", "c")
```

Comment:
My pet peeve, here and elsewhere:

```suggestion
    d = c("a", "b", "c"),
```